content/en/blog/_posts/2019-11-05-kubernetes-with-microk8s.md

```diff
@@ -0,0 +1,149 @@
+---                                           
+title: 'Running Kubernetes locally on Linux with Microk8s'                                                           
+date: 2019-11-05
+---
+**Authors**: [Ihor Dvoretskyi](https://twitter.com/idvoretskyi), Developer Advocate, Cloud Native Computing Foundation; [Carmine Rimi](https://twitter.com/carminerimi), Product Manager, Canonical
+
+This is article the second in a series about local deployment options on Linux, and it will cover [MicroK8s](https://microk8s.io/)  -the click-and-run solution for deploying a Kubernetes cluster locally, originally developed by Canonical, the publisher of Ubuntu.
+
+While Minikube usually spins-up a local virtual machine (VM) for the Kubernetes cluster, MicroK8s doesn’t require a VM. It uses [snap](https://snapcraft.io/) packages, an application packaging and isolation technology (also originally developed by Canonical).
```

Comment:
nit: do you need the hyphen spins-up: While Minikube usually spins up a ...
nit: do you need the phrase (also originally developed by Canonical)